amlip_py/amlip_py/node/FiwareNode.py

```diff
@@ -169,8 +169,12 @@ def run(self):
         Start the Flask server.
 
         """
-
-        self.app.run(host=self.server_ip, port=self.server_port)
+        try:
+            self.app.run(host=self.server_ip, port=self.server_port)
```

Comment:
You shouldn’t catch the exception here because `self.app.run()` is blocking, and catching exceptions in this block prevents them from propagating to higher levels where you might need to act on them.

Reply:
Done